pkgs/applications/networking/instant-messengers/zoom-us/default.nix

```diff
@@ -44,23 +44,23 @@ let
   throwSystem = throw "Unsupported system: ${system}";
 
   # Zoom versions are released at different times for each platform
-  version = {
-    aarch64-darwin =import ./arm64-darwin-version.nix;
+  versions = {
+    aarch64-darwin = import ./arm64-darwin-version.nix;
     x86_64-darwin = import ./x86_64-darwin-version.nix;
     x86_64-linux = import ./x86_64-linux-version.nix;
-   }.${system} or throwSystem;
+  };
```

Comment:
Why did you change this?

Reply:
I changed it so that all three attributes of `srcs` would have valid URLs. That helped me test that all three URLs are downloadable and have the right hash. Otherwise, two of the three attributes of `srcs` will be spurious URLs that may give 404 errors since they have the wrong version number for that platform (and in fact that is exactly what happens with the current state of the Zoom website).